production

```diff
@@ -41,6 +41,6 @@ WORKDIR /app
 RUN apt-get clean && \
     apt-get autoremove -y
 
-RUN bundle install
+RUN gem install bundler && bundle install
```

Comment:
ok!

Reply:
So, is it possible to move these `development` and `production` dockerfiles into a folder called `dockerfiles` or something? They're just a little mysterious sitting in the root folder. I don't recall we have them this way in plots2- but for comparison i think we have docker compose files in this directory:

https://github.com/publiclab/plots2/tree/main/containers/

Reply:
I made some code suggestions for `dockerfile/development` etc below which you can accept, then we just need to move the files themselves. Almost there!!!